brew/stop-uninstall-wipe.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+set -u # also -e?
```

Comment:
```suggestion
set -e
set -u
set -o pipefail
```
https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425